### ISSUE TITLE
Is there any plan to support subscription?

### ISSUE THREAD

**Issue by lynic** [shurcooL/graphql#27]

**Comment by dmitshur** [comment]
Yes, it is a long term goal. However, I don't plan to work on this soon. It's not needed for [githubv4](https://github.com/shurcooL/githubv4) because the GitHub GraphQL API v4 doesn't make use of the feature.

If you have a need for this feature, I would suggest developing it in a fork. Once it's ready for general consumption, I'd be happy to eventually pull it in.

**Comment by tkallioras** [comment]
@dmitshur is there any chance that the solution by @adamgoose be added in the package?

**Comment by rajdas98** [comment]
Hi all, Any update on this issue?

**Comment by KlausVii** [comment]
https://github.com/hasura/go-graphql-client seems to provide subscriptions
